src/containers/OrderReviewPage.jsx

```diff
@@ -243,6 +248,14 @@ class OrderReviewPage extends React.Component {
 
   render() {
     const { orderData, isLoading } = this.state;
+    const isValid = (orderData && orderData._order[0]._deliveries && orderData._order[0]._billingaddressinfo && orderData._order[0]._paymentmethodinfo);
```

Comment:
In cases of digital assets, shipping address doesn't need to be present. I'll fix this before merging.